src/client/helpers.js

```diff
@@ -193,14 +193,13 @@ function checkRequestResponse(response, result) {
   }
 
   // The following occurs on delete requests
-  if (response.statusCode === 204 && response.statusMessage === 'No Content') {
-    return createError(response.statusMessage, response.statusCode);
+  if (response.status === 204 && response.statusText === 'No Content') {
+    return createError(response.statusText, response.status);
```

Comment:
@teebot I also needed to update this and the code below as part of the new fetch api, just slight changes :)